server/src/tracing.ts

```diff
@@ -38,10 +38,10 @@
 
 import EventEmitter from "node:events";
 import * as Sentry from "@sentry/node";
-import { Transaction } from "@sentry/tracing";
-import type { Span, SpanStatusType } from "@sentry/tracing";
+import { Transaction } from "@sentry/core";
+import type { Span, SpanStatusType } from "@sentry/core";
 import type { SpanContext } from "@sentry/types";
-export type { Span } from "@sentry/tracing";
+export type { Span } from "@sentry/core";
```

Comment:
*Technically* we should probably add `@sentry/core` to the `package.json` file because of this… but I’m not going to. We’ll break in CI if an update causes that package to be unavailable, so I think we’re OK.

*Not* specifying it gets us out of the stupid mess where all the Sentry packages have to use the exact same version number, but Dependabot doesn’t know that, leading to Dependabot PRs that always fail, and which someone has to manually fix. Avoiding that feels like a win.